notebooks/crop_area_estimation.ipynb

```diff
@@ -9,20 +9,20 @@
     "\n",
```

Comment:
<p>Personal preference: if functions are declared inside Jupyter notebooks they should be as close as possible to the cell where they are called to reduce scrolling if modifying the function</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nasaharvest/crop-mask/pull/207/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>